securesignals-ima/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+}
+
+apply from:  rootProject.file("$rootDir/common.gradle")
+
+android {
+    namespace 'com.uid2.securesignals.ima'
+    defaultConfig {
+        minSdk 19
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+}
+
+dependencies {
+    implementation project(path: ':sdk')
+
+    // We only compile against the IMA SDK as the exact version being used should be defined by the consuming
+    // application.
+    compileOnly 'com.google.ads.interactivemedia.v3:interactivemedia:3.29.0'
```

Comment:
Is it possible to a user a lower minor version?  The Plugin needs to support 3.x+ .  There shouldn't be a breaking change, but it probably wouldn't hurt to be overly conservative on this.

Reply:
As long as the interface doesn't change, yeah. We can move to a lower `compileOnly` version once we know our minimum